2022-03-14 11:05:53,303 [main] INFO com.classes.BaseTest - Loading all methods from -masterVO class
2022-03-14 11:05:53,306 [main] INFO com.utils._masterVO - Loading Properties file from: .//src//test//resources//sysconfig//Constant.properties
2022-03-14 11:05:53,309 [main] INFO com.utils._masterVO - Properties File Loaded successfully
2022-03-14 11:05:53,311 [main] INFO com.utils._masterVO - Loading Master Sheet from: src//test//resources//config//DataProvider.xlsx
2022-03-14 11:05:53,897 [main] INFO com.utils._masterVO - Master Sheet Loaded successfully
2022-03-14 11:05:53,899 [main] INFO com.utils._masterVO - Loading Client Library for Client Versioning from: .//src//test//resources//sysconfig//ClientLib.properties
2022-03-14 11:05:53,902 [main] INFO com.utils._masterVO - Client Library Loaded successfully
2022-03-14 11:05:54,169 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::ZO - AUTbKlI - AUToKa1C
2022-03-14 11:05:54,301 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::AR - AUT7dba - AUT63dCN
2022-03-14 11:05:54,428 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::SA - AUTO2Fm - AUTNelnh
2022-03-14 11:05:54,863 [main] INFO com.utils._masterVO - Loading test case file file from: .//src//test//resources//sysconfig//testCaseFile.properties
2022-03-14 11:05:54,874 [main] INFO com.utils._masterVO - Test cases file Loaded successfully
2022-03-14 11:05:54,875 [main] INFO com.utils._masterVO - Loading Web Cases Master file from: .//src//test//resources//config//testCaseFile.properties
2022-03-14 11:05:54,893 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-03-14 11:05:54,918 [main] INFO com.dbrepository.DBHandler - Entered Access Handler :: PostGreSQL
2022-03-14 11:05:54,923 [main] INFO com.commons.CacheController - New Cache DAO Loaded :: UPDATECACHE_VER = 1
2022-03-14 11:05:54,952 [MLog-Init-Reporter] INFO com.mchange.v2.log.MLog - MLog clients using log4j logging.
2022-03-14 11:05:55,027 [main] INFO com.mchange.v2.c3p0.C3P0Registry - Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
2022-03-14 11:05:55,105 [main] INFO com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 5, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge146an11fqj54ce6ocd|3fa247d1, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> org.postgresql.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge146an11fqj54ce6ocd|3fa247d1, idleConnectionTestPeriod -> 0, initialPoolSize -> 10, jdbcUrl -> jdbc:postgresql://172.30.38.120:5432/pretups_trunk, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2022-03-14 11:05:55,557 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getNetworkName()
2022-03-14 11:05:55,679 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: Network Name (Network GGN)
2022-03-14 11:05:55,680 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(AMOUNT_MULT_FACTOR)
2022-03-14 11:05:55,681 [main] INFO com.dbrepository.DBConnectionPool - Active Size:1 Cache Size:2
2022-03-14 11:05:55,755 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (100)
2022-03-14 11:05:55,757 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MSISDN_PREFIX_LENGTH)
2022-03-14 11:05:55,759 [main] INFO com.dbrepository.DBConnectionPool - Active Size:2 Cache Size:1
2022-03-14 11:05:55,831 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (2)
2022-03-14 11:05:55,832 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MAX_MSISDN_LENGTH)
2022-03-14 11:05:55,833 [main] INFO com.dbrepository.DBConnectionPool - Active Size:3 Cache Size:0
2022-03-14 11:05:55,987 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (15)
2022-03-14 11:05:55,989 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(USER_PRODUCT_MULTIPLE_WALLET)
2022-03-14 11:05:55,991 [main] INFO com.dbrepository.DBConnectionPool - Active Size:4 Cache Size:2
2022-03-14 11:05:56,064 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2022-03-14 11:05:56,066 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(EXTERNAL_DATE_FORMAT)
2022-03-14 11:05:56,067 [main] INFO com.dbrepository.DBConnectionPool - Active Size:5 Cache Size:1
2022-03-14 11:05:56,145 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (yyyy/MM/dd)
2022-03-14 11:05:56,146 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DATE_FORMAT_CAL_JAVA)
2022-03-14 11:05:56,147 [main] INFO com.dbrepository.DBConnectionPool - Active Size:6 Cache Size:0
2022-03-14 11:05:56,351 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (yyyy/MM/dd)
2022-03-14 11:05:56,352 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(OTH_COM_CHNL)
2022-03-14 11:05:56,353 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-03-14 11:05:56,428 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2022-03-14 11:05:56,429 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DW_ALLOWED_GATEWAYS)
2022-03-14 11:05:56,430 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-03-14 11:05:56,503 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:05:56,504 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_LANGUAGE)
2022-03-14 11:05:56,505 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-03-14 11:05:56,576 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (en)
2022-03-14 11:05:56,577 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_COUNTRY)
2022-03-14 11:05:56,578 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-03-14 11:05:56,653 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (US)
2022-03-14 11:05:56,655 [main] INFO com.utils._masterVO - Display Locale set to: en_US
2022-03-14 11:06:17,709 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-03-14 11:06:17,726 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//English.properties
2022-03-14 11:06:17,756 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2022-03-14 11:06:17,757 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//C2S_English.properties
2022-03-14 11:06:17,768 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2022-03-14 11:06:17,769 [main] INFO com.dbrepository.PostGreSQLRepository - Entered getGatewayDetails('EXTGW','USSD','SMSC')
2022-03-14 11:06:17,770 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-03-14 11:06:17,776 [main] INFO com.dbrepository.PostGreSQLRepository - getGatewayDetails() :: select query: SELECT rmg.gateway_code, mg.gateway_type, rmg.login_id, rmg.PASSWORD, rmg.content_type, rmg.service_port, mg.req_password_plain FROM message_gateway mg, req_message_gateway rmg WHERE mg.gateway_code = rmg.gateway_code AND rmg.gateway_code IN ('EXTGW','USSD','SMSC')
2022-03-14 11:06:17,863 [main] INFO com.dbrepository.PostGreSQLRepository - Database Returns: [[Ljava.lang.Object;@6cce16f4[{{EXTGW,EXTGW,pretups,0971500a350af5c3d1c0b12221a0558c,XML,190,Y},{USSD,USSD,pretups,0971500a350af5c3d1c0b12221a0558c,XML,190,Y},{SMSC,SMSC,pretups,0971500a350af5c3d1c0b12221a0558c,PLAIN,190,Y}}]
2022-03-14 11:06:17,864 [main] INFO com.dbrepository.PostGreSQLRepository - Exiting getGatewayDetails()
2022-03-14 11:06:17,868 [main] INFO com.utils._APIUtil - Entered getAPIServices(EXTGW)
2022-03-14 11:06:17,876 [main] INFO com.utils._APIUtil - Exiting getAPIServices([])
2022-03-14 11:06:17,879 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-03-14 11:06:18,006 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,010 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,012 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,027 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,059 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,078 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,079 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,086 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,087 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,088 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,089 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,091 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,092 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,092 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,093 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,115 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,115 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,119 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,119 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,121 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,121 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,121 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,122 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,122 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,122 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,123 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,123 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,123 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,123 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,124 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,124 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,125 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,125 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,125 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,127 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,127 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,127 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,128 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,128 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,128 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,128 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,129 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,129 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,129 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:311)
	at com.utils._masterVO.loadGatewayCases(_masterVO.java:115)
	at com.utils._masterVO.loadExternalServices(_masterVO.java:448)
	at com.utils._masterVO.loadVO(_masterVO.java:81)
	at com.classes.BaseTest.setupReporter(BaseTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:401)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)

2022-03-14 11:06:18,447 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-03-14 11:06:18,449 [main] INFO com.utils._APIUtil - Entered getAPIServices(USSD)
2022-03-14 11:06:18,452 [main] INFO com.utils._APIUtil - Exiting getAPIServices([])
2022-03-14 11:06:18,453 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-03-14 11:06:18,649 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-03-14 11:06:18,651 [main] INFO com.utils._masterVO - Loading Gateway Cases Master file from: .//src//test//resources//config//GatewayMaster.xlsx
2022-03-14 11:06:18,773 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2022-03-14 11:06:39,814 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.ConnectException: Connection timed out: connect
2022-03-14 11:06:39,844 [main] INFO com.classes.BaseTest - Completed loading all methods from -masterVO class
2022-03-14 11:08:08,523 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-03-14 11:08:08,523 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_01_Test_success()
2022-03-14 11:08:08,541 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:08:08,541 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:08:09,337 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:47,287 [main] INFO com.classes.BaseTest - java.net.SocketException: Software caused connection abort: recv failed
2022-03-14 11:29:47,296 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:48,547 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-03-14 11:29:48,547 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_01_Test_success()
2022-03-14 11:29:48,551 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:48,551 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:48,560 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:48,604 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:48,612 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:49,063 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-03-14 11:29:49,063 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_01_Test_success()
2022-03-14 11:29:49,064 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:49,064 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:49,068 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:49,083 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:49,087 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:49,524 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-03-14 11:29:49,525 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_01_Test_success()
2022-03-14 11:29:49,528 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:49,528 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:49,532 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:49,550 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:49,553 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:50,083 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_01_Test_success -------------------
2022-03-14 11:29:50,083 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_01_Test_success()
2022-03-14 11:29:50,085 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:50,085 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:50,089 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:50,120 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:50,126 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:50,732 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_02_PriorDate -------------------
2022-03-14 11:29:50,733 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_02_PriorDate()
2022-03-14 11:29:50,743 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:50,743 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:50,749 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:50,776 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:50,780 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:51,331 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_02_PriorDate -------------------
2022-03-14 11:29:51,331 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_02_PriorDate()
2022-03-14 11:29:51,335 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:51,335 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:51,340 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:51,355 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:51,360 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:51,894 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_02_PriorDate -------------------
2022-03-14 11:29:51,894 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_02_PriorDate()
2022-03-14 11:29:51,895 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:51,896 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:51,899 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:51,915 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:51,918 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:52,381 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_02_PriorDate -------------------
2022-03-14 11:29:52,381 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_02_PriorDate()
2022-03-14 11:29:52,383 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:52,383 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:52,388 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:52,408 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:52,412 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:52,941 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_02_PriorDate -------------------
2022-03-14 11:29:52,941 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_02_PriorDate()
2022-03-14 11:29:52,943 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:52,943 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:52,949 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:52,963 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:52,970 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:53,691 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_03_DateWithScheduleNow -------------------
2022-03-14 11:29:53,691 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_03_DateWithScheduleNow()
2022-03-14 11:29:53,693 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:53,693 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:53,696 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:53,713 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:53,720 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:54,243 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_03_DateWithScheduleNow -------------------
2022-03-14 11:29:54,243 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_03_DateWithScheduleNow()
2022-03-14 11:29:54,245 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:54,245 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:54,262 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:54,276 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:54,280 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:54,796 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_03_DateWithScheduleNow -------------------
2022-03-14 11:29:54,797 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_03_DateWithScheduleNow()
2022-03-14 11:29:54,799 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:54,799 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:54,804 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:54,816 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:54,820 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:55,445 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_03_DateWithScheduleNow -------------------
2022-03-14 11:29:55,445 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_03_DateWithScheduleNow()
2022-03-14 11:29:55,452 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:55,452 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:55,457 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:55,472 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:55,475 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:56,062 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_03_DateWithScheduleNow -------------------
2022-03-14 11:29:56,062 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_03_DateWithScheduleNow()
2022-03-14 11:29:56,062 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:56,063 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:56,069 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:56,083 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:56,088 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:56,625 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_04_InvalidPIN -------------------
2022-03-14 11:29:56,625 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_04_InvalidPIN()
2022-03-14 11:29:56,627 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:56,627 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:56,634 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:56,658 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:56,661 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:57,256 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_04_InvalidPIN -------------------
2022-03-14 11:29:57,257 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_04_InvalidPIN()
2022-03-14 11:29:57,258 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:57,259 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:57,277 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:57,288 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:57,292 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:57,795 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_04_InvalidPIN -------------------
2022-03-14 11:29:57,795 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_04_InvalidPIN()
2022-03-14 11:29:57,796 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:57,796 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:57,800 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:57,811 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:57,814 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:58,379 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_04_InvalidPIN -------------------
2022-03-14 11:29:58,379 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_04_InvalidPIN()
2022-03-14 11:29:58,381 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:58,381 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:58,387 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:58,402 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:58,405 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:58,954 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_04_InvalidPIN -------------------
2022-03-14 11:29:58,955 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_04_InvalidPIN()
2022-03-14 11:29:58,956 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:58,956 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:58,960 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:58,970 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:58,973 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:59,559 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_05_BlankScheduleDate -------------------
2022-03-14 11:29:59,559 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_05_BlankScheduleDate()
2022-03-14 11:29:59,561 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:29:59,561 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:29:59,565 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:29:59,575 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:29:59,578 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:00,069 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_05_BlankScheduleDate -------------------
2022-03-14 11:30:00,069 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_05_BlankScheduleDate()
2022-03-14 11:30:00,071 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:30:00,071 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:30:00,074 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:30:00,085 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:00,088 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:00,692 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_05_BlankScheduleDate -------------------
2022-03-14 11:30:00,692 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_05_BlankScheduleDate()
2022-03-14 11:30:00,694 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:30:00,694 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:30:00,698 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:30:00,720 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:00,723 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:01,390 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_05_BlankScheduleDate -------------------
2022-03-14 11:30:01,390 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_05_BlankScheduleDate()
2022-03-14 11:30:01,392 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:30:01,392 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:30:01,404 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:30:01,423 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:01,427 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:02,240 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_05_BlankScheduleDate -------------------
2022-03-14 11:30:02,241 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_05_BlankScheduleDate()
2022-03-14 11:30:02,244 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2022-03-14 11:30:02,245 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered Test_OAuthenticationTest()
2022-03-14 11:30:02,252 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2022-03-14 11:30:02,270 [main] INFO com.classes.BaseTest - java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:02,277 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:03,216 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_06_BlankPIN -------------------
2022-03-14 11:30:03,216 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_06_BlankPIN()
2022-03-14 11:30:03,225 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:03,228 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 729796282086362
2022-03-14 11:30:03,231 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:03,234 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:03,237 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 721361662568124
2022-03-14 11:30:03,240 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:03,243 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:03,247 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 726070515171682
2022-03-14 11:30:03,250 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:03,254 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(C2C_BATCH_FILEEXT)
2022-03-14 11:30:03,258 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:10
2022-03-14 11:30:03,272 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-03-14 11:30:03,273 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: An I/O error occurred while sending to the backend.
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:335)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:235)
	at restassuredapi.test.C2SBulkPrepaidRecharge.setupData(C2SBulkPrepaidRecharge.java:127)
	at restassuredapi.test.C2SBulkPrepaidRecharge.A_06_BlankPIN(C2SBulkPrepaidRecharge.java:407)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.SocketException: Connection reset by peer: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(Unknown Source)
	at java.net.SocketOutputStream.write(Unknown Source)
	at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
	at java.io.BufferedOutputStream.flush(Unknown Source)
	at org.postgresql.core.PGStream.flush(PGStream.java:514)
	at org.postgresql.core.v3.QueryExecutorImpl.sendSync(QueryExecutorImpl.java:1365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	... 35 more

2022-03-14 11:30:03,279 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:30:03,288 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/c2cFileServices/downloadRCTemplate
2022-03-14 11:30:03,313 [main] INFO com.classes.BaseTest - java.lang.IllegalArgumentException: accessToken cannot be null
2022-03-14 11:30:03,318 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:04,153 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_06_BlankPIN -------------------
2022-03-14 11:30:04,154 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_06_BlankPIN()
2022-03-14 11:30:04,156 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:04,159 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 721033901202695
2022-03-14 11:30:04,163 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:04,166 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:04,170 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 723436079376307
2022-03-14 11:30:04,174 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:04,178 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:04,182 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 723758652695514
2022-03-14 11:30:04,186 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:04,190 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(C2C_BATCH_FILEEXT)
2022-03-14 11:30:04,194 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2022-03-14 11:30:04,198 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-03-14 11:30:04,199 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: An I/O error occurred while sending to the backend.
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:335)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:235)
	at restassuredapi.test.C2SBulkPrepaidRecharge.setupData(C2SBulkPrepaidRecharge.java:127)
	at restassuredapi.test.C2SBulkPrepaidRecharge.A_06_BlankPIN(C2SBulkPrepaidRecharge.java:407)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.SocketException: Connection reset by peer: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(Unknown Source)
	at java.net.SocketOutputStream.write(Unknown Source)
	at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
	at java.io.BufferedOutputStream.flush(Unknown Source)
	at org.postgresql.core.PGStream.flush(PGStream.java:514)
	at org.postgresql.core.v3.QueryExecutorImpl.sendSync(QueryExecutorImpl.java:1365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	... 35 more

2022-03-14 11:30:04,203 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:30:04,207 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/c2cFileServices/downloadRCTemplate
2022-03-14 11:30:04,214 [main] INFO com.classes.BaseTest - java.lang.IllegalArgumentException: accessToken cannot be null
2022-03-14 11:30:04,219 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:05,103 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_06_BlankPIN -------------------
2022-03-14 11:30:05,103 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_06_BlankPIN()
2022-03-14 11:30:05,107 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,111 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 726034865049098
2022-03-14 11:30:05,114 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,118 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,123 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 725553759170604
2022-03-14 11:30:05,126 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,130 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,134 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 720169605468434
2022-03-14 11:30:05,139 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,143 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(C2C_BATCH_FILEEXT)
2022-03-14 11:30:05,149 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:8
2022-03-14 11:30:05,153 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-03-14 11:30:05,155 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: An I/O error occurred while sending to the backend.
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:335)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:235)
	at restassuredapi.test.C2SBulkPrepaidRecharge.setupData(C2SBulkPrepaidRecharge.java:127)
	at restassuredapi.test.C2SBulkPrepaidRecharge.A_06_BlankPIN(C2SBulkPrepaidRecharge.java:407)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.SocketException: Connection reset by peer: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(Unknown Source)
	at java.net.SocketOutputStream.write(Unknown Source)
	at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
	at java.io.BufferedOutputStream.flush(Unknown Source)
	at org.postgresql.core.PGStream.flush(PGStream.java:514)
	at org.postgresql.core.v3.QueryExecutorImpl.sendSync(QueryExecutorImpl.java:1365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	... 35 more

2022-03-14 11:30:05,160 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:30:05,165 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/c2cFileServices/downloadRCTemplate
2022-03-14 11:30:05,173 [main] INFO com.classes.BaseTest - java.lang.IllegalArgumentException: accessToken cannot be null
2022-03-14 11:30:05,178 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:05,958 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_06_BlankPIN -------------------
2022-03-14 11:30:05,958 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_06_BlankPIN()
2022-03-14 11:30:05,963 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,966 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 729733366734947
2022-03-14 11:30:05,970 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,975 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,979 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 724883919378488
2022-03-14 11:30:05,983 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,987 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:05,990 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 724214606935434
2022-03-14 11:30:05,993 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:05,996 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(C2C_BATCH_FILEEXT)
2022-03-14 11:30:06,000 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:7
2022-03-14 11:30:06,006 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-03-14 11:30:06,006 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: An I/O error occurred while sending to the backend.
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:335)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:235)
	at restassuredapi.test.C2SBulkPrepaidRecharge.setupData(C2SBulkPrepaidRecharge.java:127)
	at restassuredapi.test.C2SBulkPrepaidRecharge.A_06_BlankPIN(C2SBulkPrepaidRecharge.java:407)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.SocketException: Connection reset by peer: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(Unknown Source)
	at java.net.SocketOutputStream.write(Unknown Source)
	at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
	at java.io.BufferedOutputStream.flush(Unknown Source)
	at org.postgresql.core.PGStream.flush(PGStream.java:514)
	at org.postgresql.core.v3.QueryExecutorImpl.sendSync(QueryExecutorImpl.java:1365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	... 35 more

2022-03-14 11:30:06,010 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:30:06,015 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/c2cFileServices/downloadRCTemplate
2022-03-14 11:30:06,022 [main] INFO com.classes.BaseTest - java.lang.IllegalArgumentException: accessToken cannot be null
2022-03-14 11:30:06,027 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
2022-03-14 11:30:06,826 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - ------------------ Test Case Execution Started For A_06_BlankPIN -------------------
2022-03-14 11:30:06,826 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Entered A_06_BlankPIN()
2022-03-14 11:30:06,829 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:06,834 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 728338253078029
2022-03-14 11:30:06,837 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:06,841 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:06,845 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 727787192548554
2022-03-14 11:30:06,849 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:06,854 [main] INFO com.utils.GenerateMSISDN - GenerateMSISDN class Returns: Remaining MSISDNLength (13)
2022-03-14 11:30:06,859 [main] INFO com.classes.UniqueChecker - The generated MSISDN is :: 727189801020250
2022-03-14 11:30:06,865 [main] INFO restassuredapi.test.C2SBulkPrepaidRecharge - Creating Data to Write.
2022-03-14 11:30:06,869 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(C2C_BATCH_FILEEXT)
2022-03-14 11:30:06,874 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:6
2022-03-14 11:30:06,878 [main] INFO com.dbrepository.PostGreSQLRepository - Exception while fetching System Preference: 
2022-03-14 11:30:06,879 [main] INFO com.dbrepository.PostGreSQLRepository - org.postgresql.util.PSQLException: An I/O error occurred while sending to the backend.
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:335)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:307)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:293)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:270)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:224)
	at com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
	at com.dbrepository.PostGreSQLRepository.getSystemPreference(PostGreSQLRepository.java:235)
	at restassuredapi.test.C2SBulkPrepaidRecharge.setupData(C2SBulkPrepaidRecharge.java:127)
	at restassuredapi.test.C2SBulkPrepaidRecharge.A_06_BlankPIN(C2SBulkPrepaidRecharge.java:407)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.SocketException: Connection reset by peer: socket write error
	at java.net.SocketOutputStream.socketWrite0(Native Method)
	at java.net.SocketOutputStream.socketWrite(Unknown Source)
	at java.net.SocketOutputStream.write(Unknown Source)
	at java.io.BufferedOutputStream.flushBuffer(Unknown Source)
	at java.io.BufferedOutputStream.flush(Unknown Source)
	at org.postgresql.core.PGStream.flush(PGStream.java:514)
	at org.postgresql.core.v3.QueryExecutorImpl.sendSync(QueryExecutorImpl.java:1365)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:306)
	... 35 more

2022-03-14 11:30:06,884 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (null)
2022-03-14 11:30:06,888 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/c2cFileServices/downloadRCTemplate
2022-03-14 11:30:06,897 [main] INFO com.classes.BaseTest - java.lang.IllegalArgumentException: accessToken cannot be null
2022-03-14 11:30:06,902 [main] ERROR com.sshmanager.SSHService - Error while getting SSH Server Instance : com.jcraft.jsch.JSchException: java.net.NoRouteToHostException: No route to host: connect
